plugins/idea/pluginImpl/src/main/java/com/imaginea/kodebeagle/impl/util/ScalaImportsUtil.java

```diff
@@ -15,6 +15,7 @@
  * limitations under the License.
  */
 
+// TODO will use for scala support
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)